src/screens.native.js

```diff
@@ -26,20 +26,33 @@ export function useScreens(shouldUseScreens = true) {
   }
 }
 
-export function screensEnabled() {
+function screensEnabled() {
   return USE_SCREENS;
 }
 
-export const NativeScreen = requireNativeComponent('RNSScreen', null);
+let NativeScreenValue;
+let NativeScreenContainerValue;
 
-const AnimatedNativeScreen = Animated.createAnimatedComponent(NativeScreen);
+const ScreensNativeModules = {
+  get NativeScreen() {
+    NativeScreenValue =
+      NativeScreenValue || requireNativeComponent('RNSScreen', null);
+    return NativeScreenValue;
+  },
 
-export const NativeScreenContainer = requireNativeComponent(
-  'RNSScreenContainer',
-  null
+  get NativeScreenContainer() {
+    NativeScreenContainerValue =
+      NativeScreenContainerValue ||
+      requireNativeComponent('RNSScreenContainer', null);
+    return NativeScreenContainerValue;
+  },
+};
+
+const AnimatedNativeScreen = Animated.createAnimatedComponent(
+  ScreensNativeModules.NativeScreen
```

Comment:
Oops I missed this. So this Animated component also needs to be created lazily since you're not lazily initialising on render, but initialising on property access. Since `ScreensNativeModules.NativeScreen` is called in the module scope, the variable will be initialised as the module is loaded and will throw if the component wasn't linked.

Reply:
I have addressed this in #117